SAMtags.tex

```diff
@@ -271,6 +273,15 @@ \subsection{Barcodes}
 A unique ID within the SAM file for the source molecule from which this read is derived. 
 All reads with the same {\tt MI} tag represent the group of reads derived from the same source molecule. 
 
+\item[CR:Z:\tagvalue{str}]
+Cellular barcode. 
+Cellular barcode sequence as reported by the sequencer (uncorrected) with the respective quality scores stored in {\tt CY}. All reads with the same {\tt CR} tag derive from the same cell. 
```

Comment:
Same issue as in the short description.  "sequence" what (bases)?

Reply:
All the above comments make sense. I sorted tags alphabetically in table 1 and in the Barcodes section. I also rephrased the description of the tags to make them more similar to the style of RX tag as suggested.